AirSkyBoat/AirSkyBoat
# Audit VCS NPC Content Tags

**_I affirm:_**

## What does this pull request do?

Audit [VCS NPC](https://ffxiclopedia.fandom.com/wiki/Vana%27diel_Chocobo_Society) content tags; includes CS NPCs as well.

## Steps to test these changes

Load a server before TOAU and check NPC locations.

Fixes https://github.com/AirSkyBoat/AirSkyBoat/issues/1172

Concludes Bastok NPC audit.
